### DISCUSSION TITLE
strongswan.conf syntax issue in 5.9.0

### DISCUSSION THREAD

**Discussion by mattscipione** [strongswan/strongswan#1239] [Q&A]
I recently had a customer install a new version of Strongswan; going from 5.1.1 all the way to 5.9.0 and ran into the following error:

/system/strongswan/etc/strongswan.conf:13: syntax error, unexpected ., expecting : or '{' or '=' [.]
invalid config file '/system/strongswan/etc/strongswan.conf'

Line 13 is as follows:

/system/strongswan/etc/charon.log

I removed the . between charon and log and ipsec then started up with no issue.  I was unable to find this error discussed on the 5.9.0 notes and strongswan.org still references the filelog as charon.log

Is this a known issue with needing to remove the . in the log file to not trigger an operator out of place error?

**Comment by tobiasbrunner** [comment]
> I was unable to find this error discussed on the 5.9.0 notes 

Because this has been changed a long time ago (with [5.7.0](https://wiki.strongswan.org/versions/70)). Perhaps you should have read the changelogs of all versions released over the nearly 10 years since 5.1.1 ;-)

> and strongswan.org still references the filelog as charon.log

Not sure what you are referring to, but see [here](https://docs.strongswan.org/docs/5.9/config/logging.html#_file_loggers).

**Comment by mattscipione** [comment]
Thanks Tobias, good to know it's expected behavior.

https://docs.strongswan.org/docs/5.9/config/logging.html#_example

This section of the 5.9 manual shows "charon.log" as the charon logpath.  Hence my confusion over the .

**Reply by tobiasbrunner** [reply]
> This section of the 5.9 manual shows "charon.log" as the charon logpath

Which is perfectly fine as it's not in the section name but the `path` setting.
